tests/metrics/workload_time/docker_workload_time.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test measures the complete workload of a container using docker.
+#  From calling docker until the workload is completed and the container
+#  is shutdown.
+
+set -e
+
+if [ $# -ne 4 ]; then
+  echo -e "Usage: $0 <cmd to run> <image> <runtime> <times to run>" && exit 1
+fi
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "$SCRIPT_PATH/../common/test.common"
+
+CMD=$1
+IMAGE=$2
+RUNTIME=$3
+TIMES=$4
+TMP_FILE=/tmp/workloadTime
+TEST_NAME="docker run time"
+TEST_ARGS="image=$IMAGE command=$CMD runtime=$RUNTIME units=seconds"
+TEST_RESULT_FILE=$(echo "$RESULT_DIR/$TEST_NAME-$IMAGE-$CMD-$RUNTIME" | sed 's| |-|g')
+
+function run_workload(){
+  if [ "$RUNTIME" == 'runc' ] || [ "$RUNTIME" == 'cor' ]; then
+    (time -p $DOCKER_EXE run -ti --runtime "$RUNTIME" "$IMAGE" "$CMD") &> "$TMP_FILE"
+    if [ $? -eq 0 ]; then
+      test_data=$(grep ^real "$TMP_FILE" | cut -f2 -d' ')
+      write_result_to_file "$TEST_NAME" "$TEST_ARGS" "$test_data" "$TEST_RESULT_FILE"
+    fi
+    rm -rf $TMP_FILE
+  else
+    echo "Runtime: $RUNTIME is not valid"
```

Comment:
Should this be an error (exit/die?) or is it just a warning? Either way, it is clearer to check `$RUNTIME` at the start of the function (using only `if ...; then ... fi`) and deal with errors there to avoid having to use `else`.